.env.sample

```
@@ -108,4 +108,5 @@ ORG_UPDATES_TOPIC="dev.organizationEvent"
 USER_ACCOUNT_EVENT_TOPIC = elevate_user_account_event_listener                             // Kafka topic to listen user account events
 SERVICE_NAME_HEALTH_CHECK = SamikshaService
 SESSION_VERIFICATION_METHOD = user_service_authenticated                        // session verification method
-USER_SERVICE_INTERNAL_ACCESS_TOKEN_HEADER_KEY = internal_access_token           // user service's internal access token header key
\ No newline at end of file
+USER_SERVICE_INTERNAL_ACCESS_TOKEN_HEADER_KEY = internal_access_token           // user service's internal access token header key
+TENANT_CACHE_TTL = 86400                                                       // ms for 24 hrs
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)